src/sentry/static/sentry/app/views/settings/components/forms/model.tsx

```diff
@@ -721,4 +722,45 @@ class FormModel {
   }
 }
 
+/**
+ * The mock model mocks the model interface to simply return values from the props
+ *
+ * This is valuable for using form fields outside of a Form context. Disables a
+ * lot of functionality however.
+ */
+export class MockModel {
+  //TODO(TS)
+  props: any;
+
+  initialData: object;
+
+  constructor(props) {
+    this.props = props;
+
+    this.initialData = {
+      [props.name]: props.value,
+    };
+  }
+
+  setValue() {}
```

Comment:
should we add descriptions here too?

Reply:
I'm OK with the mocked class not having any documentation.